frontend/src/Metamaps/DataModel/index.js

```diff
@@ -82,7 +82,7 @@ const DataModel = {
     var starredCollection = serverData.Starred ? serverData.Starred : []
     var mapperCollection = serverData.Mapper ? serverData.Mapper : []
     var mapperOptionsObj = { id: 'mapper', sortBy: 'updated_at' }
-    if (serverData.Mapper.mapperId) {
+    if (serverData.Mapper && serverData.Mapper.mapperId) {
```

Comment:
I think this is a change I made that should be on develop. I wonder why it's showing up in this PR?